crypto-app/src/scenes/navbar/Link.tsx

```diff
@@ -0,0 +1,29 @@
+import { SelectedPage } from "@/shared/types";
+import AnchorLink from "react-anchor-link-smooth-scroll"
+
+type Props = {
+    page: string;
+    selectedPage: SelectedPage;
+    setSelectedPage: (value: SelectedPage) => void
+}
+
+const Link = ({
+    page,
+    selectedPage,
+    setSelectedPage,
+}: Props) => {
+    const lowerCasePage = page.toLowerCase().replace(/ /g, "") as SelectedPage
```

Comment:
```suggestion
    const lowerCasePageText = pageText?.toLowerCase().replace(/ /g, "") as SelectedPage
```

Update lowerCasePageText usage